Entity/Revision.php

```diff
@@ -770,6 +782,15 @@ public function getContentType()
         return $this->contentType;
     }
 
+    public function getContentTypeName(): string
+    {
+        if (null === $this->contentType) {
+            throw new \Exception('No contentType for revision!');
```

Comment:
What'sbetter between \Exception and \RuntimeException in this case?

Reply:
2fe4fb7